sphinx/conf.py

```diff
@@ -389,5 +389,6 @@ def getValue(value):
     r'https://trac.openmicroscopy.org*',
     r'http://openslide.cs.cmu.edu*', # NameResolutionError
     r'https://openslide.cs.cmu.edu*', # NameResolutionError
-    r'https://tronche.com/computer-graphics/*' # NameResolutionError
+    r'https://tronche.com/computer-graphics/*', # NameResolutionError
```

Comment:
```
Wednesday 24 July 2024 at 01:00:00
Expires On	Friday 15 August 2025 at 00:59:59
```
Certificate is now valid

Reply:
fixed in https://github.com/ome/bio-formats-documentation/pull/394/commits/a3ef84ee89ed69228628ee1232a41bb80df45a72